nomic/gpt4all/gpt4all.py

```diff
@@ -23,19 +26,18 @@ def __init__(self, llama_path=None):
         self.model_path = llama_path
         self.tokenizer_path = llama_path
         self.lora_path = 'nomic-ai/vicuna-lora-multi-turn_epoch_2'
```

Comment:
wait is the gpt4all model the first model we trained? should we update this to a newer one?

Reply:
We should remove gpt4all from the repository, but lets do it in a separate PR

Reply:
Removal should definitely be part of the 2.0 release.

Reply:
🪓 done